.clang-format

```diff
@@ -0,0 +1,116 @@
+---
+# BasedOnStyle:  WebKit
```

Comment:
this looks too long and complex...

isn't possible use WebKit style and just override what we want to change?

Reply:
So you want me to commit the .clang-format from webkit and then commit the change between this one, right ?

Reply:
nope. I mean something like this: 

```
BasedOnStyle: WebKit
AlignConsecutiveAssignments: true
AlignConsecutiveDeclarations: true
```

Where you express: "Use WebKit style but change AlignConsecutiveAssignments and AlignConsecutiveDeclarations which are false in WebKit style"

Reply:
you can create your own format here https://clang-format-configurator.site/

Reply:
that's what I did I should maybe change it but it is mostly based on WebKit style.